frontend/src/pages/Home.tsx

```diff
@@ -0,0 +1,455 @@
+import homeCover from "../assets/home_cover.svg";
+import Accordion from "../components/Accordion";
+import { Page } from "../components/Page";
+import { Pathway } from "../components/Pathway";
+import styles from "../stylesheets/Home.module.css";
+
+export function Home() {
+  return (
+    <Page>
+      <h1 className={styles.title}>Become a Certified Interior Designer (CID)</h1>
+      <img alt="home cover" src={homeCover} />
+      <div className={styles.textDivider}>
+        <h2 className={styles.heading}>CID Candidate Information</h2>
```

Comment:
This is now put in the center instead of right aligned. [Here is the figma of that page](https://www.figma.com/file/cvxkabfqRGSE6t4S1lhiYM/CCIDC-Design-Work?node-id=1548%3A19856&mode=dev)